py_src/ipyelk/tools/tools.py

```diff
@@ -26,12 +26,21 @@ def _default_description(self):
         return "Toggle Collapsed"
 
     def handler(self, *args):
-        diagram: ElkDiagram = self.app.diagram
         transformer: XELK = self.app.transformer
         graph, tree = transformer.source
-        for element_id in diagram.selected:
+        for element_id in self.app.selected:
             if element_id in tree:
                 for child in tree.neighbors(element_id):
                     state = tree.nodes[child].get("hidden", False)
                     tree.nodes[child]["hidden"] = not state
                 self.app.refresh()
+
+
+class FitBtn(ToolButton):
+    @T.default("description")
+    def _default_description(self):
+        return "Fit"
+
+    def handler(self, *args):
+        diagram: ElkDiagram = self.app.diagram
```

Comment:
inline annotations have a smell... it is _possible_ to hoist traitlets with e.g.
```
class Foo(HasTraits):
  bar = Unicode(allow_none=True)  # type: Optional[Text]
```

if we're going to be serious about this, we need to add e.g. mypy...

Reply:
Added followon issue https://github.com/jupyrdf/ipyelk/issues/25